.circleci/config.yml

```diff
@@ -436,6 +415,68 @@ jobs:
           path: /tmp/results.html
       - store_artifacts:
           path: /tmp/results.json
+
+  cloud-e2e-k8s-idpe:
+    machine:
+      # use Ubuntu 16 image as jsonnet requires libraries newer than in Ubuntu 14
+      image: ubuntu-1604:202007-01
+    # E2e tests deploy kind and are very resource intensive.
+    # 2xlarge is the largest resource class for "machine"; there is no 2xlarge+ like the docker runners.
+    resource_class: 2xlarge
+    working_directory: ~/
+    steps:
+      - checkout:
+          path: ./ui
+      - install_jsonnet_binary
+      - install_kind_binary
+      - install_kubectl_binary
+      - install_kubecfg_binary
+      - attach_workspace:
+          at: /tmp/workspace
+      - checkout_k8s_idpe
+      - checkout_idpe
+      - checkout_monitor_ci
+      - run:
+          name: Make test artifact directories
+          command: |
+            mkdir -p /tmp/artifacts
+            mkdir -p /tmp/cypress_artifacts
+      - run:
+          name: Copy over the testing variables
+          command: cp ./monitor-ci/env.testing.cloud ./monitor-ci/.env
+      - run: docker login -u=$QUAY_USER -p=$QUAY_PASS quay.io
+      - run:
+          name: Build ui-e2e image
+          command: |
+            cd ./monitor-ci && make build NODE=cypress
+      # TODO: enable this when we remove cloud-e2e job
```

Comment:
we dont want to push these images unless the tests pass, which doesn't seem to be what this is saying. it's fine to be wasteful of the ${CIRCLE_SHA1} tags, but definitely get the latest tag out of here, or else any changes to the tests... even if they're broken... get sent directly to the next idpe PR

Reply:
like master should have already been tested.. but this effectively says "dont test the merge" right? and we should either be more explicit about that, or move this

Reply:
huh yeah I see what you're saying but it's interesting because this is what cloud-e2e is doing currently, right? https://github.com/influxdata/ui/blob/master/.circleci/config.yml#L189

Reply:
I can move both of them to be executed after the tests run since that makes more sense